### PR TITLE
Create+cluster

### PR DESCRIPTION
**Pull request by alan-wu**
Update to version 1.6.0 of ZincJs.

Changelog here: https://github.com/alan-wu/ZincJS/blob/develop/CHANGELOG.md

ScaffoldVuer:

Add new interface for create points and lines.
Add option to enable/disable marker cluster.
Now supports lines with LOD.
Improve primitives settings interface.

### CODE REVIEW

File: src/components/ScaffoldVuer.vue [hunk 1]
```diff
@@ -2095,6 +2447,20 @@ export default {
...
+
+  &:focus {
+    border: none;
+    outline: none;
+  }
+
+  &.shape {
+    margin-left: 4px;
+    margin-right: 4px;
+    color: var(--el-color-primary-light-5);
```
Comment (ddjnw1yu) [inline src/components/ScaffoldVuer.vue:2459]: Two color here need to add `!important`. Otherwise, the tool icon style will be overwritten by mapvuer.

**Review by ddjnw1yu — APPROVED** [review]
The functionality looks all good.

File: src/components/ScaffoldVuer.vue [hunk 2]
```diff
@@ -2095,6 +2447,20 @@ export default {
...
+    &.active {
+      color: var(--el-color-primary);
```
Comment (ddjnw1yu) [inline src/components/ScaffoldVuer.vue:2461]: Here

File: src/components/ScaffoldVuer.vue (hunk 1 above)
Comment (alan-wu) [reply src/components/ScaffoldVuer.vue]: Thanks for spotting this.

File: src/components/ScaffoldVuer.vue (hunk 2 above)
Comment (alan-wu) [reply src/components/ScaffoldVuer.vue]: Thanks for spotting this.

**Pull request merged**